doc/api/fs.md

```diff
@@ -7509,6 +7509,19 @@ added:
 
 Free blocks available to unprivileged users.
 
+**Example:**
+```javascript
```

Comment:
```suggestion
```mjs
```

Reply:
Could you please verify all the changes?